### PR TITLE
[DOCS-6155] SAP Cloud Connector docs

### PR DESCRIPTION
**Pull request by anxumalo**

### CODE REVIEW

File: sap-cloud/latest/admin/index.md [hunk 1]
```diff
@@ -2,3 +2,92 @@
...
+
+Follow the guidance to test the newly installed and configured module for selected SAP Business Objects.
+
+### Material Master
+
+Upload an attachment to a Material Master so it's shown in the **Business Object Types > Products** folder in Content Services.
+
+1. Go to transaction `MM03` and choose any material:
+
+   ![SAP transaction MM03]({% link sap-cloud/images/sap_test_mm03_1.png %}
```
Comment (oliveralfresco2018) [inline sap-cloud/latest/admin/index.md:17]: Looks like the images arent showing on mine.

Comment (oliveralfresco2018) [reply sap-cloud/latest/admin/index.md:17]: All of them :-)

File: sap-cloud/latest/admin/index.md [hunk 2]
```diff
@@ -2,3 +2,92 @@
...
+| Equipment | `EQUI` | `IE03` | :heavy_check_mark: |
+| Inbound Delivery | `BUS2015` | `VL33N` | :heavy_check_mark: |
+| Material Master | `BUS1001006` | `MM03` | :heavy_check_mark: |
+| Outbound Delivery | `LIKP` | `VL03` | :heavy_check_mark: |
+| Product | `MARA`, `BUS1001001`, `BUS1001006` | `MM03` | :heavy_check_mark: |
+| Production Order  | `PORDER` | `CO03` | :heavy_check_mark: |
+
+## Troubleshooting
+
+Sorry you're having trouble with the Content Connector for SAP Cloud.
```
Comment (oliveralfresco2018) [inline sap-cloud/latest/admin/index.md:67]:
```suggestion
If you're having trouble with the Content Connector for SAP Cloud.
```

**Review by oliveralfresco2018 — COMMENTED** [review]
All looks ok to me, except the images arent showing when i build it I just see the image location.

File: sap-cloud/latest/admin/index.md (hunk 2 above)
Comment (anxumalo) [reply sap-cloud/latest/admin/index.md:67]: No change

**Pull request merged**